packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -401,6 +402,20 @@ export const schema: SchemaType<DbTag> = {
     editableBy: ['sunshineRegiment', 'admins'],
     insertableBy: ['sunshineRegiment', 'admins'],
   },
+  
+  postsDefaultSortOrder: {
+    type: String,
+    optional: true,
+    group: formGroups.advancedOptions,
+    viewableBy: ['guests'],
+    editableBy: ['sunshineRegiment', 'admins'],
+    insertableBy: ['sunshineRegiment', 'admins'],
+    control: 'select',
+    options: () => Object.entries(SORT_ORDER_OPTIONS).map(([key, val]) => ({
```

Comment:
Can you return it to the default after you've set it? Even if so, I would strongly consider adding "Most Relevant" as an option to this list.

Reply:
Yup, you can clear this field to return it to the default. I'll add "Most Relevant" though, just in case we ever change the default.